.github/workflows/ci-cd.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         arch: [amd64, arm64]
     steps:
       - name: Install go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
```

Comment:
In this specific place please disable caching: see https://github.com/orgs/community/discussions/50687

Reply:
fixed